Brokerages/Tradier/TradierBrokerage.cs

```diff
@@ -183,7 +183,10 @@ public T Execute<T>(RestRequest request, TradierApiRequestType type, string root
 
                 if (!raw.IsSuccessful)
                 {
-                    throw new Exception($"TradierBrokerage.Execute(): Status: {raw.StatusCode}, Response: {raw.Content}");
+                    throw new Exception("TradierBrokerage.Execute(): " +
+                        $"Status: {raw.StatusCode}, " +
+                        $"Response: {raw.Content}, " +
+                        $"Error: {(raw.ErrorException != null ? raw.ErrorException.ToString() : string.Empty)}");
```

Comment:
Think just doing `Error: {raw.ErrorException}` works correctly, null is handled as an empty string